jest.config.js

```diff
@@ -0,0 +1,48 @@
+module.exports = {
+  clearMocks: true,
+
+  collectCoverage: false,
+
+  coverageDirectory: 'coverage',
+
+  coverageProvider: 'v8',
+
+  moduleFileExtensions: [
+    'js',
+    'mjs',
+    'cjs',
+    'jsx',
+    'ts',
+    'tsx',
+    'json',
+    'node'
+  ],
```

Comment:
```suggestion
  moduleFileExtensions: [
    'js',
    'jsx',
    'ts',
    'tsx',
  ],
```

저희한테 필요한 부분만 해줘도 될 것 같습니다!